.github/workflows/ci.yml

```diff
@@ -30,6 +30,18 @@ jobs:
           cache: "yarn"
       - run: yarn install
       - run: yarn run format-check
+  rubocop:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Install Ruby and Bundler
+        uses: ruby/setup-ruby@v1
+        with:
+          bundler-cache: true
+          ruby-version: '3.0'
```

Comment:
We should probably use a `.ruby-version file for this so it is consistent with local dev.

Also we should remove `TargetRubyVersion` from this repo's `.rubocop.yml` so that Rubocop can pick up `.ruby-version` too - see https://docs.rubocop.org/rubocop/configuration.html#setting-the-target-ruby-version

Reply:
🐘 ideally I'd prefer this change be done in its own PR :)

> Also we should remove TargetRubyVersion from this repo's .rubocop.yml

I don't think it matters that much, but fwiw I thought the comment on why that is in there made sense ("Target the oldest version of Ruby that Rails 7 supports")